samples/helmexample/README.md

```diff
@@ -0,0 +1,136 @@
+# How-To: Deploy to AKS using Helm
+
+## Setup Resource Group
```

Comment:
Non-blocking idea: Add the az CLI options or download the arm template to create such an environment